json-spec/examples/CBERS_4_MUX_20170528_090_084_L2.json

```diff
@@ -51,55 +51,59 @@
   "links": {
     "self": {
       "rel": "self",
-      "href": "https://cbers-stac.s3.amazonaws.com/CBERS4/MUX/090/084/CBERS_4_MUX_20180622_090_084_L2.json"
+      "href": "https://cbers-stac.s3.amazonaws.com/CBERS4/MUX/090/084/CBERS_4_MUX_20170528_090_084_L2.json"
     },
     "catalog": {
       "rel": "catalog",
       "href": "https://cbers-stac.s3.amazonaws.com/CBERS4/MUX/090/catalog.json"
     },
-    "cx:collection": {
+    "collection": {
       "rel": "collection",
       "href": "https://cbers-stac.s3.amazonaws.com/collections/CBERS_4_MUX_L2_collection.json"
     }
   },
   "assets": {
     "thumbnail": {
-      "href": "https://s3.amazonaws.com/cbers-meta-pds/CBERS4/MUX/090/084/CBERS_4_MUX_20180622_090_084_L2/CBERS_4_MUX_20180622_090_084.jpg",
+      "href": "https://s3.amazonaws.com/cbers-meta-pds/CBERS4/MUX/090/084/CBERS_4_MUX_20170528_090_084_L2/CBERS_4_MUX_20170528_090_084.jpg",
       "required": true,
       "type": "jpeg"
     },
     "metadata": {
-      "href": "s3://cbers-pds/CBERS4/MUX/090/084/CBERS_4_MUX_20180622_090_084_L2/CBERS_4_MUX_20180622_090_084_L2_BAND6.xml",
+      "href": "s3://cbers-pds/CBERS4/MUX/090/084/CBERS_4_MUX_20170528_090_084_L2/CBERS_4_MUX_20170528_090_084_L2_BAND6.xml",
       "required": true,
       "type": "xml"
     },
     "B5": {
-      "href": "s3://cbers-pds/CBERS4/MUX/090/084/CBERS_4_MUX_20180622_090_084_L2/CBERS_4_MUX_20180622_090_084_L2_BAND5.tif",
+      "href": "s3://cbers-pds/CBERS4/MUX/090/084/CBERS_4_MUX_20170528_090_084_L2/CBERS_4_MUX_20170528_090_084_L2_BAND5.tif",
       "type": "GeoTIFF",
+      "format": "COG",
       "required": true,
```

Comment:
This is an example of what I mentioned in the gitter forum, it is not clear to me the difference between 'type' and 'format'. Also, if we should use 'COG' for cloud optimized and 'GetTIFF' for not optimized.

Reply:
Yeah, we need to just set down some recommendations for this. Discussion in https://github.com/radiantearth/stac-spec/issues/54